bimmer_connected/all_trips.py

```diff
@@ -101,6 +101,16 @@ def __getattr__(self, item):
         """Generic get function for all backend attributes."""
         return self._state.attributes[SERVICE_ALL_TRIPS][item]
 
+    @property
+    def available_attributes(self) -> List[str]:
+        """Get the list of all-trips attributes available for this vehicle."""
+        result = ['average_combined_consumption', 'average_electric_consumption',
+            'average_recuperation', 'battery_size_max', 'chargecycle_range',
+            'reset_date', 'saved_co2', 'saved_co2_green_energy', 
+            'total_electric_distance', 'total_saved_fuel'
+        ]
```

Comment:
Are all these attributes allways available for a car which has `all_trips`?
Same question of course for the `charging_profile` and `last_trips`?

Reply:
Yep, as far as I know yes. I only have an I3 and mine has; and I checked the responses in the test-folder also...